src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -312,12 +363,37 @@ val qualityFormat: Option[FastqQualityFormat] = None,
     // Close the writer; NB: the inputs close automatically
     sampleInfos.foreach(_.writer.close())
 
+    // Optionally create the FASTQs
+    if (outputFastqs) {
+      logger.info("Writing FASTQs")
+      val fastqProgress = new ProgressLogger(this.logger, unit=1e6.toInt)
```

Comment:
If things stay this way you probably want to give different noun/verbs here so that it's clear what the progress is for.